app/incident/views.py

```diff
@@ -35,7 +35,7 @@ def get(self, request):
         except Exception as e:
             return JsonResponse({
                 'message': MESSAGES[lng]
-                .get('ERROR_MESSAGE_INVALID_TOKEN').format(str(e)),
+                .get('ERROR_MESSAGE_FOR_INCIDENT_CATEGORY').format(str(e)),
```

Comment:
I don't think this .format is needed now

Reply:
Remove all unwanted occurances of .format, use only where message has '{}' in string i.e. 

```json
"ERROR_MESSAGE_INVALID_DATA": "Invalid data provided: {}",
"ERROR_MESSAGE_NOT_FOUND": "Resource not found: {}",
```

Reply:
For now its fine I'll mark this one as resolved